onnxruntime/test/providers/cpu/tensor/slice_op.test.cc

```diff
@@ -548,7 +562,7 @@ TEST(SliceTest, Slice2D_ReverseAllAxes) {
   RunSliceTest<float>({2, 2},
                       {1.0f, 2.0f, 3.0f, 4.0f},
                       {-1, -1},
-                      {std::numeric_limits<int64_t>::max(), std::numeric_limits<int64_t>::max()},
+                      {std::numeric_limits<int64_t>::min(), std::numeric_limits<int64_t>::min()},
```

Comment:
note: using INT_MAX end when stepping backwards doesn't play well with ONNX shape inference which the CoreML EP relies on. the ONNX spec now suggests using INT_MIN end when slicing backwards so just update to match that. there's still one test that covers the INT_MAX end while stepping backwards case.